API-source/models/todolists.php

```diff
@@ -56,40 +56,40 @@ public static function show_all($logged_in_user, $list_id = NULL) {
 
     public static function display_todos($logged_in_user, $todolist_id) {
```

Comment:
If todolist is a teamgantt todolist, we sync it first, then return the list as normal.